js/web/script/test-runner-cli.ts

```diff
@@ -167,6 +167,7 @@ async function main() {
       cpuOptions: args.cpuOptions,
       webglOptions: args.webglOptions,
       webnnOptions: args.webnnOptions,
+      qnnOptions: args.qnnOptions,
```

Comment:
no need to put QNN options in ort-web tests as it is not going to support QNN in browser. should remove all changes safely under folder /js/web/

Reply:
removed changes under /js/web per request